### DISCUSSION TITLE
Add badges to headers (or text)

### DISCUSSION THREAD

**Discussion by pdimens** [facebook/docusaurus#5477] [Ideas]
Hello,

Is there a way to provide inline badges/tags of some kind to headers or body text? The use case may be pointing out a deprecation in a section, or specifying that a section is documenting a plugin from a different repo, etc. I looked through the `remark` library and didn't see anything that fit the bill, however I'm not really a web developer so maybe I overlooked it.

**Comment by slorber** [comment]
I don't understand what you mean here. 

Please show an example of what you want from another site/doc, or draw some mock-ups and give a more concrete example with real label/data/use-case.

If you want to render a badge, you can create your own React-based badge and use it in md docs thanks to MDX

**Comment by pdimens** [comment]
Here is a mockup of adding a simple "experimental" badge (or tag?) to a header
![image](https://user-images.githubusercontent.com/19176506/132024184-f4a865b4-dfd2-4d18-9af8-28adeb1130c9.png)

**Reply by slorber** [reply]
The ReactNative site does it:

https://reactnative.dev/docs/next/accessibilityinfo#getrecommendedtimeoutmillis-android

![image](https://user-images.githubusercontent.com/749374/132027618-982ffb16-b7f4-4f78-9be1-6c3126a6808d.png)


Not sure how though, maybe @Simek can tell you ;)

**Reply by Simek** [reply]
The implementation if quite simple, we just place `div` with the text and custom `class` in the header line:
* https://github.com/facebook/react-native-website/blame/main/docs/accessibilityinfo.md#L189

Then we use CSS (with pseudo-elements) to style it:
* https://github.com/facebook/react-native-website/blob/main/website/src/css/customTheme.scss#L1120
 
"Tags" added this way will also be visible in the ToC, so you need to style them too in there or hide them out, if it's not what you are looking for.

<img alt="Screenshot 2021-09-03 173008" src="https://user-images.githubusercontent.com/719641/132030607-ab71a4e5-a412-490a-bc20-83843a048c4a.png">

**Reply by slorber** [reply]
Thanks :) that looks like a nice solution, wonder how we could make it easier without introducing some new fancy md syntax
